src/main/java/ru/mambetpages/controller/PersonalProfileController.java

```diff
@@ -0,0 +1,46 @@
+package ru.mambetpages.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.*;
```

Comment:
You should avoid star imports. @varya-kot create checkstyle issue, i guess using maven plugin.